resources/kcp/charts/kyma-environment-broker/templates/trial-cleanup-job.yaml

```diff
@@ -15,7 +15,7 @@ spec:
           {{ end }}
         spec:
           shareProcessNamespace: true
-          {{- with .Values.deployment.securityContext }}
+          {{- with .ValuesAPP_DRY_RUN.deployment.securityContext }}
```

Comment:
I think this won't render as `.ValuesAPP_DRY_RUN`

Reply:
Thanks - it was accidental change.

Reply:
Reverted in the next commit